front/pages/w/[wId]/builder/assistants/dust.tsx

```diff
@@ -77,17 +87,31 @@ InferGetServerSidePropsType<typeof getServerSideProps>) {
     (c) => c.name === "dust"
   );
   if (!dustAgentConfiguration) {
-    void router.push(`/w/${owner.sId}/builder/assistants`);
     return null;
   }
 
+  let defaultAgentName: string | null = null;
+  for (const agentSid of DEFAULT_ASSISTANTS_ORDER) {
+    const agentConfig = agentConfigurations?.find((c) => c.sId === agentSid);
+    if (agentConfig && agentConfig.status === "active") {
+      defaultAgentName = agentConfig.name;
+      break;
+    }
+  }
+
   const handleToggleAgentStatus = async (agent: AgentConfigurationType) => {
     if (agent.status === "disabled_free_workspace") {
       window.alert(
         `@${agent.name} is only available on our paid plans. Contact us at team@dust.tt to get access.`
       );
       return;
     }
+    if (agent.status === "disabled_missing_datasource") {
+      window.alert(
```

Comment:
Let’s use the new alerts?

Reply:
actually it's unreachable, but still moving this + the other window.alert to the notif system 👍